docs/guides/digital-signature.md

```diff
@@ -1,31 +1,31 @@
 # Digital signature
 
-Certifaction's digital signature allows client to add one or more digital signatures to PDF documents.
+Certifaction's digital signature allows users to add one or more digital signatures to PDF documents.
 
-Certifaction document signature is based on the [PAdES](https://en.wikipedia.org/wiki/PAdES) standard,
-which uses cryptography and the public PKI infrastructure to provides the highest guarantee against forgery.
+Certifaction document signatures are based on the [PAdES](https://en.wikipedia.org/wiki/PAdES) standard,
+which uses cryptography and Public Key Infrastructure (PKI) to provide the highest guarantee against forgery.
 
 Certifaction uses three signature levels with increasing identity verification assurances:
 
 - Standard Electronic Signature (SES): email verification
 - Professional Electronic Signature (PES): identification using ID document scanning and verification
 - Qualified Electronic Signature (QES): Swiss and EU qualified electronic signature requirements
 
-Certifaction's unique privacy-first signature ensures that document content does not leave your
-IT infrastructure in clear text without your consent.
+Certifaction's unique privacy-first signature ensures that the document content does not leave your
```

Comment:
"that the document content" --> "that document content" because it is generic content and not a specific content.